src/modules/crud.js

```diff
@@ -0,0 +1,37 @@
+export const createTask = (description, tasks) => {
+  const newTasks = [
+    ...tasks,
+    { index: tasks.length, description, completed: false },
+  ];
+  return newTasks;
+};
+
+export const deleteTasks = (tasks) => {
+  // Remove completed
+  const newTasks = [...tasks].filter((task) => !task.completed);
+
+  // Update indices
+  newTasks.forEach((task, index) => {
+    task.index = index;
+  });
+  return newTasks;
+};
+
+export const updateTask = (index, desc, tasks) => {
+  const newTasks = [...tasks].map((task) => {
+    if (task.index === index) task.description = desc;
+    return task;
+  });
+  return newTasks;
+};
+
+export const removeTask = (index, tasks) => {
+  // Remove completed
+  const newTasks = [...tasks].filter((task) => task.index !== index);
+
+  // Update indices
+  newTasks.forEach((task, index) => {
+    task.index = index;
+  });
+  return newTasks;
+};
```

Comment:
- [ ] When reordering the index value of each item in the array after deleting a task, kindly start from number `1`. It would make it consistent with the requirements as described in this comment https://github.com/naanmohammed/todolist-es6-webpack/pull/2#discussion_r1160341356. Remember that indexes in JavaScript in a loop start from the number `0`. Kindly implement this change. 👍 😎